src/build.js

```diff
@@ -89,6 +90,19 @@ const buildJsonSchemaPlugin = {
   },
 };
 
+const resolvePlugin = {
+  name: "resolve",
+  setup(build) {
+    console.log("setup");
+    build.onResolve(
+      { filter: /^jsonc-parser$/ },
+      async ({ path, ...options }) => {
+        return build.resolve("jsonc-parser/lib/esm/main.js", options);
```

Comment:
This package offers ESM and UMD. If I import the UMD in VSCode (the default), it makes the extension crash - if I import the ESM in the tests, the tests crash.
This way, the tests get UMD and the extension bundles with the ESM import.